docs/examples.rst

```diff
@@ -141,3 +141,14 @@ One form of running this example could be:
 .. explicit external hyperlink targets
 
 .. _examples: https://github.com/LLNL/UnifyFS/tree/dev/examples/src
+
+Transfer API
+------------
+
+UnifyFS has a transfer API to move files from the UnifyFS file space out to external storage (or from external storage into the UnifyFS file space).  This can be invoked using the "transfer" application in the examples directory.  The transfer source code also serves as a template for using transfer API through C functions.
```

Comment:
Suggest: "... to move files from UnifyFS to external storage (or from external storage into UnifyFS)." I am not sure if the terminology "file space" will confuse people

Reply:
"This can be invoked using the "transfer" application in the examples directory." Suggest to change this to: "The transfer functionality can be invoked by using the "transfer" application in the examples directory as show in the following example: " <then put the srun command srun -N4 -n4...>

Then follow with "The transfer application source code also serves as a template for using the UnifyFS transfer API in a C program."

Reply:
No objections to these changes.  

I think at some point we do need terminology in the documentation that refers to files that exist inside Unify vs. out on conventional disk.  I don't think that's clear at all in the initial pages of the documentation introduction.  But yes, let's definitely try not to confuse things here.